src/mobile/src/ui/views/wallet/Home.js

```diff
@@ -305,44 +290,35 @@ class Home extends Component {
      * Mark the task of displaying snapshot transition modal as done
      */
     completeTransitionTask() {
-        // Just mark this task as done
-        // Since most likely the account needs
-        // no transition
         this.props.markTaskAsDone({
             accountName: this.props.selectedAccountName,
             task: 'hasDisplayedTransitionGuide',
         });
+        if (this.props.isModalActive) {
```

Comment:
I think this should rather be done directly in `componentDidMount`. It doesn't really relate to `completeTransitionTask`.